MapAnalyzer/utils.py

```diff
@@ -27,7 +63,7 @@ def get_sets_with_mutual_elements(list_mdchokes: List[PathLibChoke],
     for c in list_mdchokes:
         s2 = c.pixels
         s3 = s1 ^ s2
-        if len(s3) != (len(s1) + len(s2)):
+        if len(s3) <= 0.95*(len(s1) + len(s2)):
```

Comment:
I remember we discussed this change,  but i forgot why :)
 
please comment explaining why a factor of 0.95 was added here

Reply:
The code is run to check if the detected chokes overlap with ramps or vision blockers. Previously if they had any overlap at all they got thrown out, but there can be legitimate chokes that start from near a ramp for example and have a few pixels in common so I added a small threshold. Probably could come up with some better criteria but it seems to work okay in practice